client/blocks/reader-post-card/gallery.jsx

```diff
@@ -14,7 +14,7 @@ import cssSafeUrl from 'lib/css-safe-url';
 import { isFeaturedImageInContent } from 'lib/post-normalizer/utils';
 import ReaderPostCardExcerpt from './excerpt';
 
-const GALLERY_ITEM_THUMBNAIL_WIDTH = 420;
+const READER_CONTENT_WIDTH = 800;
```

Comment:
That one confused me too. We use 720, but cards can definitely be 800 with the redesign.